sound/soc/intel/boards/skl_hda_dsp_generic.c

```diff
@@ -172,23 +172,23 @@ static int skl_hda_fill_card_info(struct snd_soc_acpi_mach_params *mach_params)
 
 static void skl_set_hda_codec_autosuspend_delay(struct snd_soc_card *card)
 {
-	struct snd_soc_pcm_runtime *rtd =
-		list_first_entry(&card->rtd_list,
-				 struct snd_soc_pcm_runtime, list);
-	struct snd_soc_dai *codec_dai = rtd->codec_dai;
```

Comment:
this is going to conflict with upstream, this was removed yesterday.

Reply:
@plbossart Ack, my bad. By using asoc_rtd_to_codec() in the original patch, would have avoid this (and need for the compile fix you made)

Reply:
just add my patch first and yours on top, and only send the second upstream.

Reply:
@plbossart Ack, I'll wait for the CI to run and if no issues, I'll send upstream